RNApp/app/routes/MapView/MapView.js

```diff
@@ -3,6 +3,7 @@ import { Text, View } from 'react-native';
 import Button from '../../components/Button';
 import styles from './styles';
 import Map from './../../components/Map';
+// import { makeRequest, getCountryName } from './../../backend/tempBackend';

```

Comment:
Oh this line isn't needed (just removed it). At some point I was going to move some of the backend function calls to MapView but in the end decided not to.